marker_finder/MarkerBoundingBox[] objects